examples.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/russross/blackfriday"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	verbose bool
+	skipped bool
+)
+
+var specsByMediaType = map[string]string{
+	"application/vnd.oci.image.manifest.v1+json":      "schema/image-manifest-schema.json",
+	"application/vnd.oci.image.manifest.list.v1+json": "schema/manifest-list-schema.json",
```

Comment:
eh? I don't follow.

@stevvooe it does make me think the schema files could be named more aligned with the mime type